invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -217,16 +217,43 @@ def _is_uri(uri):
         return False
 
 
+class URLSchema(Schema):
+    """Schema for the URL schema."""
+
+    url = SanitizedUnicode(
+        validate=_valid_url(_('Not a valid URL.'))
+    )
+
+
 class RightsSchema(Schema):
     """License schema."""
 
     id = SanitizedUnicode()
-    title = SanitizedUnicode()
-    description = SanitizedUnicode()
+    title = fields.Dict()
+    description = fields.Dict()
```

Comment:
This is WIP, need to make these 2 fields more strict and only accept locale values in this format:
```
{
   title:
       <locale>: "...",
       <other_locale>: "..."",
        ....
}
```

Reply:
About this I am not very sure how to do it @lnielsen .

In one hand if we accept any locale, we will need to check if it's valid in the backend and when visualizing it in the frontend we will need to check for the default_locale and if it's not there we will display any value inside, since we don't know the locale used to save it. 

Other option could be only use the `BABEL_DEFAULT_LOCALE` config variable to save and display the values with this locale. Potential problems could happen if we change the value since there will be `rights` values saved in 2 different locales and when displaying we will only look for the current `BABEL_DEFAULT_LOCALE` value.

Last and less problematic option is to have a hardcoded structure like:
```
{
   title:
       en : "..."
}
```
Using a harcoded locale that cannot be changed by the user we only need to check for this in the backend and frontend.

Unfortunately I cannot think of a better solution for this (I don't really like any), any suggestions are welcome.